dwds/test/build_daemon_evaluate_test.dart

```diff
@@ -204,6 +217,76 @@ void main() async {
         });
       });
 
+      test('evaluate expression in _testPackage/test_library', () async {
+        await onBreakPoint(isolate.id, testLibraryScript, 'testLibraryFunction',
+            () async {
+          var event = await stream.firstWhere(
+              (Event event) => event.kind == EventKind.kPauseBreakpoint);
+
+          var result = await service.evaluateInFrame(
+              isolate.id, event.topFrame.index, 'formal');
+
+          expect(
+              result,
+              const TypeMatcher<InstanceRef>().having(
+                  (instance) => instance.valueAsString, 'valueAsString', '23'));
+        });
+      });
+
+      test(
+          'evaluate expression in a class constructor in _testPackage/test_library',
+          () async {
+        await onBreakPoint(
+            isolate.id, testLibraryScript, 'testLibraryClassConstructor',
+            () async {
+          var event = await stream.firstWhere(
+              (Event event) => event.kind == EventKind.kPauseBreakpoint);
```

Comment:
[nit] no need to put the type on the function literal
```suggestion
              (event) => event.kind == EventKind.kPauseBreakpoint);
```

https://dart.dev/guides/language/effective-dart/design#avoid-annotating-inferred-parameter-types-on-function-expressions

Reply:
will change in the source, thanks!

Reply:
done!